### PR TITLE
Initial Creation of Azure.Health.Deidentification Dataplane SDK

### PR DESCRIPTION
**Pull request by GrahamMThomas**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by GrahamMThomas** [comment]
Related: https://github.com/Azure/azure-rest-api-specs/pull/29012

**Review by jsquire — COMMENTED** [review]
Common area files look good.   Please review the code with your team.  In the meantime, I'll kick pipeline creation and an initial set of test runs.

**Comment by jsquire** [comment]
You need a tests.yml file as well as CI.yml

**Review by danielszaniszlo — APPROVED** [review]

File: sdk/healthdataaiservices/Azure.Health.Deidentification/samples/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-health-deidentification
```
Comment (scottaddie) [inline sdk/healthdataaiservices/Azure.Health.Deidentification/samples/README.md:7]: I've emailed the Learn Information Architecture team about adding this value to the allow list

Comment (GrahamMThomas) [reply sdk/healthdataaiservices/Azure.Health.Deidentification/samples/README.md]: Pivoted to using HDS slug. Resolving

azure-health-data-services

**Review by jsquire — APPROVED** [review]

**Pull request merged**